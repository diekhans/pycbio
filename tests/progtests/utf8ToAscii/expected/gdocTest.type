output/gdocTest.txt: ASCII text
