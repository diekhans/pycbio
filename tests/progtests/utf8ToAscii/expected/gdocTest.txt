________________

MK 38672
NAXOS
RACHMANINOFF CONCERTO 2-LICAD
MDK 44908
MAHLER: SYMPHONY NO. 4 BATTLE-MAAZEL
CBS
CBS MASTERWORKS
MOZART: Piano Concertos Nos. 21 "Elvira Madigan", 12 and 14 8.550202
SEVILLA TARREGA, TORROBA, ALBENIZ TIMO KORHONEN, GUITAR
ONDINE ODE 752-2
CD-EMX 2152 COM 7 63262 2
TCHAIKOVSY 1812 OVERTURE ROMEO AND JULIET OVERTURE MARCHE SLAVE FRANCESCA DA RIMINI RLPO/EDWARDS
NAXOS
SERRA: Orchestral Works
8.555871
MUSSORGSKY PICTURES AT AN EXHIBITION-Chicago S. O./Jarvi
Chandos CHAN 8849
E2 79154
BRAHMS: PIANO PIECES, OPP. 76 & 119/FANTASIES, OP, 116
RICHARD GOODE, PIANO
ELEKTRA
NONESUCH
CDX 5009 A GOTTSCHALK FESTIVAL
VoxBox
NORRINGTONPHONIE FANTASTIQUE
CDC
Dave Grusin The Gershwin Connection
GRD-2005
7 49541 2
Contains TWO CD's With COMPLETE PROGRAM NOTE BOOKLET
VoxBox
DIGITAL STEREO
SHOSTAKOVICH: SYMPHONIE NR. 15 OKTOBER U.A. GOTHENBURG SYMPHONY ORCHESTRA/JARVI
427 616-2 N
NIGHT MOODS
DELOS
D/CD 3030
CAROL ROSENBERGER, PIANO
ORIGINAL DIGITAL
HAYDN-THE LONDON SYMPHONIES NOS. 93, 99 & 100 PHILHARMONIA ORCHESTRA - SLATKIN
09026-68425-2
HANDEL MESSIAH DAVIS
PHILIPS
438 356-2
DUO
SK 45830
BERLINER PHILHARMONIKER BARENBOIM
SONY CLASSICAL
po 999 882-2 Braunfels Phantastische Erscheinungen RSO Wien. Davies
DMITRI SHOSTAKOVICH
24 PRELUDES AND FUGUES OP.87
PAISIELLO: IL BARBIERE DI SIVIGLIA
HCD 12525-26-2
2 disc
KEITH JARRETT
PIANO
PAISIELLO: IL BARBIERE DI SIVIGLIA
HCD 12525-26-2
TCHAIKOVSKY: MANFRED SYMPHONY: MUTI
CDC 7 47412 2
ECM NEW SERIES 1469/70 437 189-2
MCGLINN
Angel" JEROME KERN TREASURY
0777 7 54883 2 6
EMILY MITCHELL FRIENDS FEATURING ARTURO DELMONI
Mozart variations
JMR 24
437 189-2
BEETHOVEN: PIANO SONATAS, VOLUME I JOHN O'CONOR, PIANO
TELARC CD-80118
JULIAN BREAM-MUSIC OF SPAIN, VOL. 4-THE CLASSICAL HERITAGE RCD1-4549
